tslint-config/react.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Didn't we say we'll have separate packages for react, so we don't install react dependencies for someone that doesn't need them?